tests/src/test/java/alluxio/client/fs/FreeAndDeleteIntegrationTest.java

```diff
@@ -110,8 +110,6 @@ public void freeAndDeleteIntegration() throws Exception {
     // Verify the block has been removed from all workers.
     assertTrue(blockInfo.getLocations().isEmpty());
     assertFalse(bw.getBlockStore().hasBlockMeta(blockId));
-    // Verify the removed block is added to LostBlocks list.
-    assertTrue(bm.isBlockLost(blockInfo.getBlockId()));
```

Comment:
If it's because the LostFileDetector has handled this block already, let's disable the LostFileDetector instead of changing the test.